shopfloor/models/shopfloor_menu.py

```diff
@@ -71,6 +114,63 @@ class ShopfloorMenu(models.Model):
         compute="_compute_allow_force_reservation_is_possible"
     )
 
+    @api.onchange("unload_package_at_destination")
+    def _onchange_unload_package_at_destination(self):
+        # Uncheck pick_pack_same_time when unload_package_at_destination is set to True
+        # Ensure that multiple_move_single_pack is False when
+        # unload_package_at_destination is checked out
+        for record in self:
+            if self.unload_package_at_destination:
```

Comment:
```suggestion
            if record.unload_package_at_destination:
```
But I don't think onchange can receive a recordset. No need of the loop on self.

Reply:
This one has been introduced in another pr https://github.com/OCA/wms/pull/398